scavenger-frontend/src/components/snapshot/SnapshotTable.vue

```diff
@@ -158,6 +165,28 @@ export default {
           ElNotification.success({message: this.$t("message.snapshot.refresh-success")});
         })
     },
+    exportSnapshot(id) {
+      const fileName = `snapshot${id}.tsv`;
+      this.$http.get(`/customers/${this.customerId}/export/snapshot/${id}?fn=${fileName}`)
+        .then((response) => {
+          const csvFile = new Blob([response.data], {type: 'text/csv'});
+          const downloadLink = document.createElement("a");
+          downloadLink.download = fileName;
+          downloadLink.href = window.URL.createObjectURL(csvFile);
+          downloadLink.style.display = "none";
+          document.body.appendChild(downloadLink);
+          downloadLink.click();
+          window.URL.revokeObjectURL(downloadLink.href);
+          document.body.removeChild(downloadLink);
+        })
+        .catch(err => {
+          if (err.response.status !== 200) {
```

Comment:
There is no case for a 200 OK in the catch, so this if condition is unnecessary.

Reply:
I fixed this on commit 268889e.